docker_util/__init__.py

```diff
@@ -0,0 +1,84 @@
+import json
+from pathlib import Path
+from subprocess import PIPE, run
+from typing import List
```

Comment:
I did consider that, and specifically decided not to. All other things equal, I'd prefer to minimize the dependencies on the host system: right now, you only need Docker, a recent enough `docker-compose` binary, new enough Python, and no Python packages beyond the standard library. I could be convinced otherwise, though.

Reply:
That's reasonable. Particularly for a commandline tool I think this is fine. (If it were inside the web application I would have security and speed concerns.)